examples/nixnet/lin-signal-single-point-output.py

```diff
@@ -4,6 +4,8 @@
  This is used to demonstrate a signal single point output session. 
  This example uses hardcoded signal names that use the NIXNET_example database. 
  Also ensure that the transceivers are externally powered when using C Series modules.
```

Comment:
Same comment about "Also ensure...". It should be after you talk about database editor.

Reply:
Done